extension-localization/README.md

```diff
@@ -0,0 +1,52 @@
+## Mapbox Maps Localization Extension for Android
+
+### Overview
+
+The Mapbox Localization PlugExtensionn for Android is an public library for localizing labels in a MapView.
+This extension library can localize the supported languaged in [mapbox-streets-v8](https://docs.mapbox.com/vector-tiles/reference/mapbox-streets-v8/#common-fields) and [mapbox-streets-v7](https://docs.mapbox.com/vector-tiles/reference/mapbox-streets-v7/#name-fields)
+
+A full overview of classes and interfaces can be found in our [API documentation](https://docs.mapbox.com/android/beta/maps/guides/).
```

Comment:
Link for API seems to be incorrect, correct one is `https://docs.mapbox.com/android/beta/maps/api-reference/`